.github/workflows/build-preview.yml

```
@@ -0,0 +1,100 @@
+      - 'README_ASSETS/**'
+      - 'test/**'
+      - '*.md'
+      - '.prettierrc.yml'
+
+jobs:
+  upload-preview:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-16.04, macos-latest, windows-latest]
+      fail-fast: false
+    name: Upload Preview
+    steps:
+    # Preperation
+    - uses: actions/checkout@v1
+    - name: Use Node.js 12.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+    - name: Get Pullrequest ID
+      id: prepare
+      run: |
+        node -e "console.log('::set-output name=prid::' + '${{ github.ref }}'.split('/')[2])"
+    - name: change build name and Product Name to development id
+      run: |
+        node ./bin/github-actions/devbuild.js
+    # Building JS
+    - name: npm install, build, test
+      run: |
+        npm install
+        npm run build
+        npm run pack:generate_config
+    # Packaging and processing
+    - name: LINUX electron builder
+      if: runner.os == 'Linux'
+      run: npx electron-builder --publish never --linux AppImage
+    - name: LINUX renaming / upload Preperation
+      if: runner.os == 'Linux'
+      run: |
+        cd dist
+        mkdir preview
+        mv *.AppImage preview/deltachat-desktop-${{ steps.prepare.outputs.prid }}.AppImage
+        ls preview
+        cd ..
+    - name: MAC electron builder
+      if: runner.os == 'macOS'
+      run: |
+        echo "module.exports=()=>Promise.resolve()" > stub.js
+        export CSC_IDENTITY_AUTO_DISCOVERY=false
+        npx electron-builder --publish never --mac mas dmg -c.mac.identity=null --config.afterSign="stub.js" || true
+    - name: MAC renaming / upload Preperation
+      if: runner.os == 'macOS'
+      run: |
+        mkdir dist/preview
+        zip -r dist/preview/deltachat-desktop-mas-${{ steps.prepare.outputs.prid }}.zip dist/mas
+        cd dist
+        mv *.dmg preview/deltachat-desktop-${{ steps.prepare.outputs.prid }}.dmg || true
+        cd ..
+        ls dist/preview
+    - name: WINDOWS electron builder
+      if: runner.os == 'Windows'
+      run: npx electron-builder --win portable --config.artifactName="deltachat-desktop-${{ steps.prepare.outputs.prid }}.portable.exe"
+    - name: WINDOWS renaming / upload Preperation
+      if: runner.os == 'Windows'
+      run: |
+        cd dist
+        dir
+        mkdir preview
+        move deltachat-desktop-${{ steps.prepare.outputs.prid }}.portable.exe preview/deltachat-desktop-${{ steps.prepare.outputs.prid }}.portable.exe
+        dir preview
+        cd ..
+    # Upload Step
+    - name: upload folder
+      id: upload
+      shell: bash
+      run: |
+        echo -e "${{ secrets.KEY }}" >__TEMP_INPUT_KEY_FILE
+        chmod 600 __TEMP_INPUT_KEY_FILE
+        scp -o StrictHostKeyChecking=no -v -i __TEMP_INPUT_KEY_FILE -P "22" -r dist/preview/ "${{ secrets.USERNAME }}"@"download.delta.chat":"/var/www/html/download/desktop/preview/"
```

```suggestion
        scp -o StrictHostKeyChecking=no -v -i __TEMP_INPUT_KEY_FILE -P "22" -r dist/preview/* "${{ secrets.USERNAME }}"@"download.delta.chat":"/var/www/html/download/desktop/preview/"
```

Probably this does the trick?